diff -Naur nut-2.8.1/common/common.c nut-2.8.1-esxi/common/common.c
--- nut-2.8.1/common/common.c	2023-10-30 00:02:33.000000000 +0100
+++ nut-2.8.1-esxi/common/common.c	2023-11-01 16:24:02.249343816 +0100
@@ -355,9 +355,6 @@
 		if (seteuid(0))
 			fatal_with_errno(EXIT_FAILURE, "getuid gave 0, but seteuid(0) failed");
 
-	if (initgroups(pw->pw_name, pw->pw_gid) == -1)
-		fatal_with_errno(EXIT_FAILURE, "initgroups");
-
 	if (setgid(pw->pw_gid) == -1)
 		fatal_with_errno(EXIT_FAILURE, "setgid");
 
